versioned_docs/version-22.04/developer/developer-gorgone-migrate-from-centcore.md

```diff
@@ -0,0 +1,98 @@
+---
+id: developer-gorgone-migrate-from-centcore
+title : Migrating from Centreon centcore
+---
+
+Follow this procedure to migrate from Centreon centcore to Centreon gorgone.
+
+You need to build a configuration file based on **/etc/centreon/conf.pm**.
+
+- If using package, execute the following command line:
+
+```shell
+$ perl /usr/local/bin/gorgone_config_init.pl
+2019-09-30 11:00:00 - INFO - file '/etc/centreon-gorgone/config.yaml' created success
+```
+
+- If using sources, execute the following command line:
+
+```shell
+$ perl ./contrib/gorgone_config_init.pl
+2019-09-30 11:00:00 - INFO - file '/etc/centreon-gorgone/config.yaml' created success
+```
+
+The following configuration file will be created at **/etc/centreon-gorgone/config.yaml**:
```

Comment:
It may be a good idea to recommend to users to change the sections according to their own initial database, config and network setup.